archive/imo/imo2013_q5.lean

```diff
@@ -142,7 +143,7 @@ begin
   { simp only [pow_one] },
   have hpn' := hpn pn.succ_pos,
   rw [pow_succ' x (pn + 1), pow_succ' (f x) (pn + 1)],
-  have hxp : 0 < x := zero_lt_one.trans hx,
+  have hxp : 0 < x := by positivity,
```

Comment:
I think this may be too simple of a proof for the tactic to be worthwhile, unless it's a really fast tactic.